src/platform/windows/drop_handler.rs

```diff
@@ -90,16 +94,25 @@ impl FileDropHandler {
             });
         });
         drop_handler.hovered_is_valid = hdrop.is_some();
+        if drop_handler.hovered_is_valid {
```

Comment:
I think it would be cleaner to express this using a single assignment:
```rust
drop_handler.cursor_effect = if drop_handler.hovered_is_valid {
    DROPEFFECT_COPY
} else {
    DROPEFFECT_NONE
};
```